extensions/wikia/WikiFactory/Close/maintenance.php

```diff
@@ -509,6 +515,28 @@ private function log( $message ) {
 		Wikia::log( "CloseWiki", false, $message, true, true );
 	}
 
+	private function removeDiscussions( int $cityId ) {
+		try {
+			$this->getSitesApi()->hardDeleteSite( $cityId, F::app()->wg->TheSchwartzSecretToken );
+		}
+		catch ( \Swagger\Client\ApiException $e ) {
+			$this->log( "{$cityId} Failed to hard delete Discussion site: {$e->getMessage()} \n" );
```

Comment:
How about we use `WikiaLogger::instance()->error()` directly here? It looks like the the `log` method calls `Wikia::log` which is deprecated, and also logs at the debug level to Kibana which means Nordic Cat won't help alert us if there's errors here.

Also, how about pulling `$cityId` and `$e->getMessage()` into separate markers? :)